guides/lambda-configuration.md

```diff
@@ -0,0 +1,68 @@
+# AWS Lambda-specific configuration
+
+This covers how to configure and enable AppOptics APM for nodejs lambda functions. For
+general configuration options see `configuration.md` in the same directory.
+
+## Prerequisite
+
+Add the `appoptics-node` layer to your function
+- screen shots?
+- select `arn:aws:lambda:${your-function's-region}:858939916050:layer:appoptics-node`:41
+- forwarder setup already done (where to document this - each agent?)
+
+choose `Specify an ARN` and select `....` - how to find/see available `appoptics-node`
+versions for node? where to see?
+
+## no-code change instrumentation
+- add the environment variable `APPOPTICS_WRAP_LAMBDA_HANDLER` set to the function's current handler.
+The current handler can be found in the function's `Basic Settings` `Handler` setting, e.g.,
+`index.handler`.
+- change the function's `Basic Settings` `Handler` setting to `appoptics-auto-lambda.handler`.
+
+That's it! When lambda loads the function it will invoke AppOptics' handler which will load your function
+(specified by the env var `APPOPTICS_WRAP_LAMBDA_HANDLER`) and automatically set up your function's instrumentation.
+
+## manual instrumentation
+- DO NOT change the lambda function's `Handler` setting nor create the env var `APPOPTICS_WRAP_LAMBDA_HANDLER`
+- add `const ao = require('appoptics-apm');` as the first module required in your function's code.
+- wrap your function before exporting it.
+
+for example, the lambda function's `Basic Settings` `Handler` is `index.handler` and your current code
+is in the file `index.js` and looks like:
+
+```js
+
+async function myHandlerEchoEvent (event, context) {
+  return {statusCode: 200, body: JSON.stringify(event)};
+}
+
+module.exports.handler = myHandlerEchoEvent;
+```
+
+to instrument make the following changes:
+
+```js
+// require the agent - contained in appoptics-node layer
+const ao = require('appoptics-apm');
+
+async function myHandlerEvent (event, context) {
+  return {statusCode: 200, body: JSON.stringify(event)};
+}
+
+// wrap user handler and export the wrapped function.
+module.exports.handler = ao.wrapLambdaHandler(myHandlerEchoEvent);
+```
+
+## even more manual installation
```

Comment:
again, not sure what you mean here.

Reply:
this paragraph
"- create your own layer that includes the appoptics-apm code? this is complicated as it involves building the bindings agent in an amazon linux container. punt."

Reply:
yes, i'm going to remove it. good question about "is the doc a work in progress?" because the answer is yes and i'd kind of forgotten about it. i'm making some edits and keeping it locally changed until some of the issues are decided.